File: assignment 10 - C# practice problems/28.cs
//Write a c# function to format a number up to specified decimal places.

using System;
namespace MyNameSpace
{
    class Program28
    {
        public static void formatNumberDriverFunction()
        {

            Console.WriteLine("Enter a float Number: ");
            var numberInput = decimal.Parse(Console.ReadLine());
            Console.WriteLine("Enter a decimal place to format: ");
            var formatDecimals = int.Parse(Console.ReadLine());

            var numberFormatted = decimal.Round(numberInput, formatDecimals, MidpointRounding.AwayFromZero);
            Console.WriteLine($"Number formatted to " + formatDecimals + " format Decimals  is " + numberFormatted);


        }

    }
}

File: assignment 3 - js practice problems/web-dev-starter/28.js
//Write a JavaScript function to format a number up to specified decimal places.
function formatNumber(numberParam, decimalPlaceParam) {

    numberParam = numberParam.toString();
    numberParam = numberParam.split(".");
    numberParam[1] = numberParam[1].substr(0, decimalPlaceParam);
    numberParam = numberParam.join(".");
    return parseFloat(numberParam);
}

function formatNumberDriverFunction() {

    let numberInput = prompt("Enter a float number");
    let deciamlPlacesInput = prompt("Specify decimal space to format");

    numberInput = parseFloat(numberInput);
    deciamlPlacesInput = parseInt(deciamlPlacesInput);

    alert("formatted number is: " + formatNumber(numberInput, deciamlPlacesInput));
}

formatNumberDriverFunction()